docs/Conceptual/00 - Introduction.md

```diff
@@ -0,0 +1,188 @@
+
+A multi-tenanted solution, on the other hand, can be thought of as *lots of solutions, lots of users*. To the *owner* of the multi-tenanted solution, it looks conceptually like *many isolated solutions*, each of which independently serves many users. To any individual tenant, their “tenancy” looks like a single set of resources serving their users, and no-one else’s.
+
+[multi-user v. multi tenant](multi-user v multi-tenant.png)
+
+## When should a solution be multi-tenanted, not multi-user?
+A multi-tenanted solution is required when one or more groups of users have a strong requirement for *isolation* from other groups of users.
+
+Typically this isolation will manifest in one or more of the following ways:
+1. The ability for a tenant to provision, manage, and monitor a collection of resources independently of other tenants.
+1. The ability for a tenant to bring their own resources (including, for example, application code, configuration, data, cryptographic keys, policy) and integrate into their solution
+1. The ability to provide particular kinds of security boundary around their solution and control identity and access (including the ability to deny access to the entity that provides the tenant with those resources)
+1. The inability for one tenant to determine the existence or otherwise of another tenant.
+
+Note that this is not simply *data isolation*. Most data storage solutions (blob storage, SQL databases etc.) have mechanisms that allow you to control access to subsets of the data, constrained to particular identities. For example, techniques like Row Level Security (RLS), filtering, and data partitioning provide various types of security and access boundary. Multi-user systems will often use these techniques to ensure segregation of user data.
+
+A multi-tenanted system will also use these techniques to provide data segregation, but adds in other constraints from the list above. 
+
+## Tenancy as a hierarchy
+It is also very common for a multi-tenanted solution to itself be used to provide other multi-tenanted solutions. This can be a source of considerable confusion.
+
+For example, the Microsoft Azure platform is a multi-tenanted solution. 
+
+An Azure tenant has resources that are isolated in all the ways described above, and we can use those resources to build solutions which are themselves multi-tenanted.
+
+With that in mind, you can think of tenancy as a tree, with a *root tenant* which owns the multi-tenanted solution. This root tenant has zero or more *child tenants* for which it provides some (other) services.
+
+[Tenant hierarchies](Tenant hierarchies.png)
+
+Why *zero* or more children? Well – a root tenant with no children is our traditional single tenant application.
+
+### Do we live in a root universe?
+The isolation rules for tenancy mean that from the perspective of the solution deployed in that tenant it is irrelevant that any given tenant has a parent (which may itself be parent...and so on...). 
```

Comment:
Apart from the perspective of being the service provider where you my need trickle down admin permissions to deliver support (although you'd ideally want PIM like elevation, even if that elevation is from a break glass recovery account you control)

Reply:
I have caveated this with a usually (see my response to Ian's comment). And I will add this as a potential feature of the management plane.